### DISCUSSION TITLE
TCP Listener channel idle time alert

### DISCUSSION THREAD

**Discussion by ropese34** [nextgenhealthcare/connect#5258] [Q&A]
Could someone provide guidance on how to add an idle time (1 hour threshold, Monday - Friday) based on alert for a TCP listener channel? Thank you

**Comment by pacmano1** [comment]
I don't understand the question.

**Reply by ropese34** [reply]
I would like to trigger an alert if there is no traffic on the listener port after 1 hour.

**Comment by aungktun** [comment]
Do you need to trigger alert if no traffic on tcp listener in some duration
x?

In paid version, you can do that kind of alert.

**Reply by ropese34** [reply]
You can't configured an alert based on idle time in the free, open source version?

**Reply by pacmano1** [reply]
You would have write that, typically by calling the rest API.   The free alert plugin is quite limtied really.

**Reply by ropese34** [reply]
I see. Are there any code samples or examples available implemented in administrator? Thanks
